ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -300,6 +319,48 @@ snpe_subplugin::configure_instance (const GstTensorFilterProperties *prop)
           } else {
             nns_logw ("Unknown runtime (%s), set CPU as default.", options[op]);
           }
+        } else if (g_ascii_strcasecmp (option[0], "OutputTensor") == 0) {
+          /* the tensor name may contain ':' */
+          gchar *_ot_str = g_strjoinv (":", &option[1]);
+          gchar **names = g_strsplit (_ot_str, ";", -1);
+          guint num_names = g_strv_length (names);
+          outputstrListHandle = Snpe_StringList_Create ();
+          for (guint i = 0; i < num_names; ++i) {
+            if (g_strcmp0 (names[i], "") == 0) {
+              nns_loge ("Given tensor name with index %u is invalid.", i);
+            }
+            nns_logi ("Add output tensor name of %s", names[i]);
+            if (Snpe_StringList_Append (outputstrListHandle, names[i]) != SNPE_SUCCESS) {
+              nns_loge ("Failed to append output tensor name. %s", names[i]);
+            }
+          }
+          g_free (_ot_str);
+          g_strfreev (names);
+        } else if (g_ascii_strcasecmp (option[0], "OutputType") == 0) {
```

Comment:
Q: with snpe filter for v1, is it possible using tensor-filter property input/inputtype/inputname and output/outputtype/outputname?

Reply:
Well I don't think so. There is no API changing Input tensors. And the "TF8" type is should be distinguished from "UINT8" type (NYI).